src/main/java/com/puppycrawl/tools/checkstyle/filters/XpathFilterElement.java

```diff
@@ -70,6 +71,9 @@ public class XpathFilterElement implements TreeWalkerFilter {
     /** Xpath query. */
     private final String xpathQuery;
 
+    /** Indicates if all properties are set to Null. */
+    private final boolean allPropertiesAreNull;
```

Comment:
please rename to something like `isEmptyConfig`

Reply:
Done